### PR TITLE
Adding image tag to example command - Closes #398

### LINKED ISSUE #398: [doc] 6.3.2.9  Configure Path to Container Images
### Type of report
   - [x] bug
   - [ ] feature request

### Source Document
6.3.2.9  Configure Path to Container Images:

https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-configure-imagepath

### Summary

We **must** not use the `:latest` tag, as it breaks the upgrade to future maintenance updates. Unfortunately, when omitting an explicit image tag, the `:latest` tag will be **implicitly** used.

Therefore, we have to provide an example using a image tag. Like

```
ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph:7.0
```

or something. In any case, we have to **avoid** the implicit `:latest`. 

### Notes
GitHub issue in ceph-salt: https://github.com/ceph/ceph-salt/issues/280

### PR DESCRIPTION
**Pull request by asettle**
Also adds a note about the usage of the image tag.

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -592,7 +605,7 @@ o- time_server ................................................ [enabled]
      If there is no default path set or your deployment requires a specific
      path, add it as follows:
     </para>
-<screen>&prompt.smaster;ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph</screen>
+<screen>&prompt.smaster;ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph:7.0</screen>
```
Comment (asettle) [inline xml/deploy_cephadm.xml:608]: This depends on internal issue

**Comment by asettle** [comment]
@sebastian-philipp any updates on this one?

**Comment by asettle** [comment]
Waiting on SES-2017

File: xml/deploy_cephadm.xml (hunk 1 above)
Comment (sebastian-philipp) [inline xml/deploy_cephadm.xml:608]:
```suggestion
<screen>&prompt.smaster;ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph:15.2.3.455.3.373</screen>
```

**Pull request closed without merging**